src/osversion/test/VersionTranslator_GTest.cpp

```diff
@@ -1397,10 +1397,12 @@ TEST_F(OSVersionFixture, update_3_1_0_to_3_2_0_CoilCoolingWaterToAirHeatPumpEqua
   // Field before: Rated COP
   EXPECT_EQ(4.2, coil.getDouble(10).get());
 
+  // 3.4.0 to 3.5.0: (3) fields added
```

Comment:
Yeah, it's weird having to modify an older test. Ideally we'd just stop at the version we were originally targeting so we don't have to do that kind of stuff